packages/office-addin-manifest/test/manifests/manifest.json

```diff
@@ -1,120 +1,82 @@
 {
   "$schema": "https://raw.githubusercontent.com/OfficeDev/microsoft-teams-app-schema/preview/DevPreview/MicrosoftTeams.schema.json",
-  "id": "df2fa15f-332a-47aa-87c2-e30c20686eee",
-  "manifestVersion": "m365DevPreview",
+  "id": "aaaaaaaa-bbbb-cccc-dddd-eeeeeeeeeeee",
   "version": "1.0.0",
+  "manifestVersion": "m365DevPreview",
   "name": {
-    "short": "My Office Add-in",
-    "full": "My Office Add-in"
+    "short": "Contoso Task Pane Add-in",
+    "full": "Contoso Task Pane Add-in"
   },
   "description": {
     "short": "A template to get started.",
-    "full": "A template to get started."
+    "full": "This is the template to get started."
   },
-  "icons": {
-    "outline": "test/assets/icon-32.png",
-    "color": "test/assets/icon-32.png"
-  },
-  "accentColor": "#FFFFFF",
   "developer": {
     "name": "Contoso",
     "websiteUrl": "https://www.contoso.com",
     "privacyUrl": "https://www.contoso.com/privacy",
-    "termsOfUseUrl": "https://www.contoso.com/terms"
+    "termsOfUseUrl": "https://www.contoso.com/servicesagreement"
```

Comment:
Can we add supportUrl here as well (https://www.contoso.com/help)?

Reply:
What is the exact name of this property?
I didn't find anymore URLs other than the ones here, so I didn't find a `supportUrl` field

Reply:
"supportUrl": "https://www.contoso.com/help"

That's the one used in the adobe example in that repo and by the conversion tool.

Reply:
Apparently, this `supportUrl` is now in the schema, I was looking at a little older version. So I just added it

Also, as FYI:
From what I understand, we shouldn't base ourselves on the Adobe example. It is not up to date
The most up-to-date example is always the alpha example and that is what I'm basing myself on.
The schema should also be up to date.